net-proxy/trojan/metadata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>yangmame@icloud.com</email>
```

Comment:
Add gentoo-proxy-maintainers project last

Reply:
Yes.

Reply:
I've finished it, please merge it